### ISSUE TITLE
pkg distribution which does not prompt for password?

### ISSUE THREAD

**Issue by Ajeey** [electron-userland/electron-builder#2175]
* **Version**: 
"electron": "1.7.8",
"electron-builder": "19.8.0",

* **Target**: 
.pkg for MAC

How to create a pkg distribution which does not prompt for password.
Tried the following options in the but no luck.
```
"pkg": {
      "allowAnywhere": false,
      "allowCurrentUserHome": true,
      "allowRootDirectory": false
    },
```

**Comment by Ajeey** [comment]
more details here - https://github.com/electron-userland/electron-builder/issues/2201

**Comment by josteph** [comment]
There is no detail available on #2201, the issue became stale and sadly closed by BOT.

Is there something that can be done in `electron-builder` level? This is quite a severe UX issue for enterprise consumers, a big comparison to using `.dmg` as installer.
